docs/src/examples/neural_sde.md

```diff
@@ -76,28 +76,24 @@ neural SDE with diagonal noise layer function:
 drift_dudt = Lux.Chain(ActivationFunction(x -> x.^3),
                        Lux.Dense(2, 50, tanh),
                        Lux.Dense(50, 2))
-p1, st1 = Lux.setup(rng, drift_dudt)
 
 diffusion_dudt = Lux.Chain(Lux.Dense(2, 2))
-p2, st2 = Lux.setup(rng, diffusion_dudt)
 
-p1 = Lux.ComponentArray(p1)
-p2 = Lux.ComponentArray(p2)
-#Component Arrays doesn't provide a name to the first ComponentVector, only subsequent ones get a name for dereferencing
-p = [p1, p2]
 
 neuralsde = NeuralDSDE(drift_dudt, diffusion_dudt, tspan, SOSRI(),
```

Comment:
The choice of the solver here is leading to https://github.com/SciML/DiffEqFlux.jl/runs/7005695604?check_suite_focus=true#step:5:22
We'll either need a different sensealg for NeuralDSDE(currently InterpolatingAdjoint) which is compatible with Lux or change the solver